### PR TITLE
Changes for the OL 20.0.0.12-beta blog

### PR DESCRIPTION
**Pull request by austin0**
A few minor changes requested by Emily J.

Signed-off-by: Austin Bailey <Austin.Bailey@ibm.com>

### CODE REVIEW

File: posts/2020-10-27-microprofile40-jakartaee9-beta-200012.adoc [hunk 1]
```diff
@@ -44,9 +44,7 @@ Open Liberty 20.0.0.12-beta provides another two feature updates for MicroProfil
...
 
-Other MicroProfile 4.0 features were part of earlier betas: MicroProfile OpenAPI 2.0 in link:{url-prefix}/blog/2020/09/29/microprofile-openapi-beta-200011.html#openAPI[20.0.0.11-beta]; MicroProfile Fault Tolerance 3.0, MicroProfile Metrics 3.0, MicroProfile Health 3.0, and MicroProfile OpenTracing 2.0 in link:{url-prefix}/blog/2020/09/02/jakarta-microprofile-beta-200010.html#allbeta[20.0.0.10-beta].
-
-While we'd hoped to also include MicroProfile Rest Client 2.0 (also part of MicroProfile 4.0), we didn't quite make it this time so it will make its debut in a future beta.
+Other MicroProfile 4.0 features were part of earlier betas; with MicroProfile Fault Tolerance 3.0, MicroProfile Metrics 3.0, MicroProfile Health 3.0, and MicroProfile OpenTracing 2.0 in the Open Liberty link:{url-prefix}/blog/2020/09/02/jakarta-microprofile-beta-200010.html#allbeta[20.0.0.10-beta], followed by MicroProfile OpenAPI 2.0 in link:{url-prefix}/blog/2020/09/29/microprofile-openapi-beta-200011.html#openAPI[20.0.0.11-beta]. While we'd hoped to also include MicroProfile Rest Client 2.0 (also part of MicroProfile 4.0), we didn't quite make it this time so it will make its debut in a future beta.
```
Comment (mbroz2) [inline posts/2020-10-27-microprofile40-jakartaee9-beta-200012.adoc:47]: I think we can add:
"However, if you just can't wait until then, you can always give the feature a try in one of our [nightly builds](https://openliberty.io/downloads/#development_builds).

**Review by mbroz2 — APPROVED** [review]

**Comment by mbroz2** [comment]
Also, I would expect to see either 2020-10-23-kerberos-grafana-200011.adoc.adoc getting renamed to 2020-10-23-kerberos-grafana-200011.adoc, or 2020-10-23-kerberos-grafana-200011.adoc.adoc being deleted.

File: posts/2020-10-27-microprofile40-jakartaee9-beta-200012.adoc (hunk 1 above)
Comment (austin0) [reply posts/2020-10-27-microprofile40-jakartaee9-beta-200012.adoc:47]: Added, will appear in next pr to avoid spamming you for approval.

Comment (mbroz2) [reply posts/2020-10-27-microprofile40-jakartaee9-beta-200012.adoc:47]: sounds good; when you do that, don't forget about the comment below as well:
"Also, I would expect to see either 2020-10-23-kerberos-grafana-200011.adoc.adoc getting renamed to 2020-10-23-kerberos-grafana-200011.adoc, or 2020-10-23-kerberos-grafana-200011.adoc.adoc being deleted."

**Comment by austin0** [comment]
@mbroz2 where do you see `2020-10-23-kerberos-grafana-200011.adoc.adoc` as I thought I had already fixed this previously. Is it just the leftover file in the draft branch?

**Comment by mbroz2** [comment]
Yes, just the one in draft

**Pull request merged**